settings.gradle.kts

```diff
@@ -1,29 +1,26 @@
-val nodeModulesMsdk = File("${rootProject.projectDir}/libs/SalesforceReact/node_modules/react-native/android") // For stand-alone MSDK builds.
-val nodeModulesTemplate = File("${rootProject.projectDir}/../../node_modules/react-native/android") // For template app builds.
-val hasNodeModules = nodeModulesMsdk.exists() || nodeModulesTemplate.exists()
-
 include("libs:SalesforceAnalytics")
 include("libs:SalesforceSDK")
 include("libs:SmartStore")
 include("libs:MobileSync")
 include("libs:SalesforceHybrid")
-if (hasNodeModules) {
```

Comment:
This condition and the variables supporting it are no longer needed since the RN artifacts are being resolved from Maven Central.